examples/kubernetesversions/kubernetes1.7.1.json

```diff
@@ -0,0 +1,36 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "orchestratorVersion": "1.7.1"
+    },

```

Comment:
I want to start removing cluster configs that provide little value—I'd opt to delete this one.